dev.edfloreshz.Calculator.json

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "dev.edfloreshz.Calculator",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.llvm18"
+  ],
+  "command": "cosmic-ext-calculator",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```

Comment:
you have to submit the exception for this.

Reply:
https://github.com/flathub-infra/flatpak-builder-lint/pull/489